pylint/lint/pylinter.py

```diff
@@ -918,7 +918,8 @@ def get_ast(
                 "syntax-error",
                 line=getattr(ex.error, "lineno", 0),
                 col_offset=getattr(ex.error, "offset", None),
-                args=str(ex.error),
+                args=f"Parsing Python code failed:\n{ex.error}",
```

Comment:
This seems unnecessary? The message is already called `syntax-error`?

Reply:
There's other syntax error for example when there's a syntax error in something we want to import. I think it does not hurt to be explicit.

Reply:
I removed the line break that could be problematic in IDE.